PythonOCC/nodes.py

```diff
@@ -110,10 +122,14 @@
     filter_points_by_distance, \
     curve_length
 
-from OCCUtils.edge import Edge
+# TODO: Edge import causes crash
+# from OCCUtils.edge import Edge
```

Comment:
This import crashes, even when I just import that in an empty python shell:

```bash
> python
Python 3.9.9 | packaged by conda-forge | (main, Dec 20 2021, 02:41:03) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from OCCUtils.edge import Edge
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<frozen zipimport>", line 259, in load_module
  File "/home/leon/apps/anaconda3/envs/pyoccenv_test/lib/python3.9/site-packages/OCCUtils-0.1.dev0-py3.9.egg/OCCUtils/edge.py", line 26, in <module>
  File "/home/leon/apps/anaconda3/envs/pyoccenv_test/lib/python3.9/site-packages/OCC/Core/GeomLib.py", line 18, in <module>
    from . import _GeomLib
ImportError: /home/leon/apps/anaconda3/envs/pyoccenv_test/lib/python3.9/site-packages/OCC/Core/_GeomLib.so: undefined symbol: _ZN7GeomLib13FuseIntervalsERK18NCollection_Array1IdES3_R20NCollection_SequenceIdEd
```

pythonocc-core version: 7.5.1